README.md

```diff
@@ -45,6 +45,21 @@ This will do the following:
 - Initialize git hooks (`npx @escape.tech/mookme init --only-hook --skip-types-selection`)
 - Install ios pods
 
+### Required Dependencies
+The following are required to run the Audius client. We recommend [homebrew](https://brew.sh/) and [pyenv](https://github.com/pyenv/pyenv).
+
+```
+brew install node nvm pyenv 
```

Comment:
Do we need node? I think nvm determines that

Reply:
So I think maybe we say you need nvm pyenv and rbenv

Reply:
But then we can `nvm i` and it will install the correct node version for us. I guess let's include that command first?